src/azure-cli/azure/cli/command_modules/network/_params.py

```diff
@@ -297,13 +297,13 @@ def load_arguments(self, _):
         c.argument('metadata', nargs='+', help='Metadata in space-separated key=value pairs. This overwrites any existing metadata.', validator=validate_metadata)
 
     with self.argument_context('network dns list-references') as c:
-        c.argument('target_resources', nargs='+', help='Space-separated list of resource IDs you wish to query.', validator=validate_subresource_list)
+        c.argument('parameters', nargs='+', help='Space-separated list of resource IDs you wish to query.', validator=validate_subresource_list)
```

Comment:
`parameters` was the actual name of the parameter, `target_resources` was just used here as a variable name but since I refactored some commands, now this is used as the actual argument name so I had to change it to `parameter` to not cause any breaking changes.